doc/api/fs.md

```diff
@@ -1536,10 +1536,10 @@ object with an `encoding` property specifying the character encoding to use.
 Example:
 
 ```js
-fs.mkdtemp('/tmp/foo-', (err, folder) => {
+fs.mkdtemp(path.join(os.tmpdir(), 'foo-'), (err, folder) => {
   if (err) throw err;
   console.log(folder);
-  // Prints: /tmp/foo-itXde2
+  // Prints: /tmp/foo-itXde2 or C:\Users\...\AppData\Local\Temp\foo-itXde2
```

Comment:
I think this is also incorrect for macOS.

Reply:
This should not be blocking though, right? As a comment it should be fine as is? (I do not have a Mac to test this on my own)

Reply:
Can confirm the tmp dir is different on macOS but ofc this comment should not block PR from landing.